website/content/docs/concepts/architecture.mdx

```diff
@@ -162,14 +176,129 @@ on hardware features such as architecture and availability of GPUs, or software
 like operating system and kernel version, or they can be business constraints like
 ensuring PCI compliant workloads run on appropriate servers.
 
+# Client Organization
```

Comment:
I'm not sure if this is the best place to have this, or if this is a good name, so I'm open to suggestions 😄

Reply:
Moved to its own page and adjusted the first section a bit:
https://github.com/hashicorp/nomad/pull/18109/files#diff-933fc372992990c21ae0cd52f2e8106d7c7c749fc34bb2617731c75477811de7R7-R19